workers/unity/Packages/com.improbable.gdk.ui/DeploymentWindow.cs

```diff
@@ -0,0 +1,186 @@
+            while (!string.IsNullOrEmpty(nextPageToken));
+
+            deployment = deployments.FirstOrDefault(dpl =>
+                dpl.Status != Deployment.Types.Status.Stopped);
+        }
+
+        public void OnGUI()
+        {            
+            using (new EditorGUILayout.VerticalScope())
+            {
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    if (GUILayout.Button("Spatial service log"))
+                    {
+                        if (Application.platform == RuntimePlatform.WindowsEditor)
+                        {                         
+                            var path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), ".improbable", "spatiald", "log", "spatiald.log");
+                            Debug.Log(path);
+                            Process.Start(path);
+                        }
+                    }
+
+                    if (GUILayout.Button("Runtime log"))
+                    {
+                        if (Application.platform == RuntimePlatform.WindowsEditor)
+                        {
+                            var path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), ".improbable", "spatiald", "log", "local", "runtime.log");
+                            Debug.Log(path);
+                            Process.Start(path);
+                        }
+                    }
+                }
+
+                if (deployment == null)
+                {
+                    if (GUILayout.Button("Local launch"))
+                    {
+                        DoLocalLaunch();
+                    }
+                }
+                else if(deployment.Status == Deployment.Types.Status.Starting)
+                {
+                    if (GUILayout.Button("Cancel"))
+                    {
+                        StopDeployment(deployment);
+                    }
+                }
+                else if (deployment.Status == Deployment.Types.Status.Running)
+                {
+                    if (GUILayout.Button("Stop"))
+                    {
+                        StopDeployment(deployment);
+                    }
+                }
+
+                if (deployment != null)
+                {
+                    GUILayout.Label($"Name: {deployment.Name}");
+                    GUILayout.Label($"Id: {deployment.Id}");
+                    GUILayout.Label($"Status: {deployment.Status}");
+                }                
+            }
+        }
+
+        private async void StopDeployment(Deployment dpl)
+        {
+            var result =
+                localDeploymentClient.StopDeploymentAsync(new StopDeploymentRequest
+                {
+                    Id = dpl.Id,
+                    ProjectName = projectName
+                });
+
+            GetDeployments();
+
+            await result;
+
+            GetDeployments();
```

Comment:
Why do we `GetDeployments`, wait for the StopDeployment command and then `GetDeployments` again?